docs/src/3.1.Worldview.mdx

```diff
@@ -8,6 +8,7 @@
 | `onCameraStateChange` | `(CameraState) => void` |                        | callback whenever the camera state changes                                                              |
 | `defaultCameraState`  | `$Shape<CameraState>`   | `DEFAULT_CAMERA_STATE` | pass in this prop to turn Worldview to an uncontrolled component. `onCameraStateChange` will be ignored |
 | `keyMap`              | `CameraKeyMap`          | {}                     | override default Worldview hotkeys                                                                      |
+| `shiftKeys`           | `boolean`               | true                   | enable/disable shift modifier for keyboard controls                                                     |
```

Comment:
nit: would be nice to alway default boolean to `false` / `undefined`.  `shiftKeys` is very straightforward to understand, how about something like `disableSlowMotionWithShiftKey`?

Reply:
Chose this name to allow possible future expansion with an object, like `shiftKeys={{ KeyA: ..., KeyS... }}`